builder/builder-next/builder.go

```diff
@@ -318,13 +319,24 @@ func (b *Builder) Build(ctx context.Context, opt backend.BuildConfig) (*builder.
 		exporterAttrs["name"] = strings.Join(opt.Options.Tags, ",")
 	}
 
+	cache := controlapi.CacheOptions{}
+
+	if inlineCache := opt.Options.BuildArgs["BUILDKIT_INLINE_CACHE"]; inlineCache != nil {
```

Comment:
Why not add `InlineCache` to `opt.Options`?

Reply:
That would mean new options in CLI and I'm not sure we are ready to make the api/cli forward compatible. After we have support for other cache strategies and containerd integration it would be safer to add inline also to that set.

Reply:
Guess we need to document the build-arg somewhere 😅 (not sure where we should put it)

Reply:
We shouldn't document it under build args, but in relation to `cache-from`